### DISCUSSION TITLE
Migrating database v1 to v2 on stopped container

### DISCUSSION THREAD

**Discussion by gTamatea** [umami-software/umami#2066] [Q&A] [answered]
Hello it might a simple question, but I'm not sure how to migrate the database. When I start the umami docker container I get 

```
✓ DATABASE_URL is defined.
✓ Database connection successful.
✓ Database version check successful.
✗ Umami v1 tables detected. For how to upgrade from v1 to v2 go to https://umami.is/docs/migrate-v1-v2.
[2K[1Gerror Command failed with exit code 1.
```

The documentation says 

```
Docker migration
Go into your running docker container. You can find the name by the output of docker ps.
docker exec -ti -u 0 <app container name> sh

Run the migration script
npx @umami/migrate-v1-v2@latest
```

But as my container exit immediately, I can't connect to it to run the command. The docker exec -ti just tells me that the container is not running. So I don't know how to execute the migrating command. Is there any solution ?

Thanks for your help

**Comment by orilights** [comment, marked as answer]
You should execute the migration script in umami v1.40.0 container, then remove the v1 container and run the v2 container.

**Reply by gTamatea** [reply]
Ok I will do that. Thanks
